### PR TITLE
Meeting minutes from the joint tech/legal call

### PR DESCRIPTION
**Pull request by goneall**

### CODE REVIEW

File: joint/2022-06-03.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+* Karsten Klein
+* Marc-Etienne Vargenau
+* Jim Vitrano
+* John Horan
+* Matija Suklje
+* Philippe Ombredanne
+* Sebastian Crane
+* Dick Brooks
+* Michael Herzog
+* Mark Atwood
```
Comment (seabass-labrax) [inline joint/2022-06-03.md:21]: I like to sort the list of attendees; it makes it a bit easier to scan for people that way. Just a matter of taste though :smiley:

File: joint/2022-06-03.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+### Topic 2 Discussion Objects:
+ 
+* Agree if this is a     problem we want to tackle 
+* Agree on outstanding     remaining issues which need to be resolved 
+* Agree on the who, what     and when for the remaining issues
+ 
+### Topic 2 Agenda:
+* Discuss the problem     statement 
+* Discuss remaining work     that needs to be done (who, what, when)
+
```
Comment (seabass-labrax) [inline joint/2022-06-03.md:49]: (this is to remove some extraneous whitespace and make the headings nest)

```suggestion
### Topic 1 – “License Namespaces”
#### Topic 1 Discussion Objectives:
* Agree on specifically which problems we plan to solve and which problems we won’t solve as it relates to referencing license texts which are not included on the SPDX license list
* For any problems we agree to solve, identify what work needs to be done, who will work on it and by when
* If we have enough time, agree on the syntax we will use for referencing external license texts
* If we have enough time, agree on the process we will implement to register and maintain external license texts
 
#### Topic 1 Agenda:
 
* Discuss specific problems we are trying to solve with namespaces and decide if the SPDX community should solve those problems (see problem list below) 10 minutes
* Discuss the policy the SPDX legal team would use as it relates to external license texts (see policy considerations below) 15 minutes
* Discuss what work needs to be done (who, what, when) 15 minutes
* Discuss syntax and process 10 minutes
 
### Topic 2 – “License Snippets in Source Files” – If there is time available
#### Topic 2 Discussion Objects:
 
* Agree if this is a problem we want to tackle
* Agree on outstanding remaining issues which need to be resolved
* Agree on the who, what and when for the remaining issues
 
#### Topic 2 Agenda:
* Discuss the problem statement
* Discuss remaining work that needs to be done (who, what, when)

```

File: joint/2022-06-03.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+  * Philippe: Take something like in Python, will have their own LicenseRefs to reference things not on SPDX list
+    * Gary: This is problem #3 below - package managers
+  * Philippe: People assume that e.g. KDE-defined LicenseRefs are stable and usable elsewhere
+  * Alexios: KDE requirements: If you use LicenseRef, must also put the text there (https://community.kde.org/Policies/Licensing_Policy#License_Texts)
+  * Does SPDX org need to solve this, or should we work with REUSE to help them solve it?
+    * Jilayne: REUSE created a way to do this; SPDX should possibly add (in Annex D?) to close the loop, recommend pointing to REUSE. Don't know what more on this particular issue we could do.
+    * Karsten: Is REUSE using LicenseRefs in this way just because that's the only option? Is there a better tool?
+    * Gary: Let's ask REUSE
+  * **Decision**: No action in SPDX specifically at this point, Will work with REUSE to align with them; if REUSE says namespace would be really valuable, would bring back to team
+  ACTION:  communicate with REUSE as per above - need to decide who will take lead on that and how to faciliate that communication
```
Comment (seabass-labrax) [inline joint/2022-06-03.md:78]:
```suggestion
  * **ACTION**:  communicate with REUSE as per above - need to decide who will take lead on that and how to facilitate that communication 
```

File: joint/2022-06-03.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+    * Karsten: we need to reflect reality, even if it's a license that is problematic; namespace allows mapping; yes, want to solve this
+    * Gary: Suggest Mark go back to Rust; if can find one package manager willing to work with us on this, then we could at least test solutions with them
+    * Sebastian: Debian's view, all packages should be buildable without an internet connection; requiring an internet connection to lookup namespaces doesn't work
+    * Philippe: existing proposal with Python on use of LicenseRef; could also with SUSE, which they prefix with "suse"
+    * Gary: open question to find a package manager / organization that's willing to publicly accept that namespaces are useful to them and willing to work with us on it
+    * Jilayne: could maybe revisit this later for the spec if and when there's someone working with us
+  * **Decision**: Don't have consensus on this yet; challenge to team to come back with package managers willing to work with us on implementing a namespace solution; if none, agree wouldn't be a problem to tackle
+  ACTION: people to discuss with package managers to get an idea on that question (in next week, if possible)
```
Comment (seabass-labrax) [inline joint/2022-06-03.md:101]:
```suggestion
  * **ACTION**: people to discuss with package managers to get an idea on that question (in next week, if possible)
```

**Review by seabass-labrax — CHANGES_REQUESTED** [review]

File: joint/2022-06-03.md (hunk 1 above)
Comment (goneall) [reply joint/2022-06-03.md:21]: @seabass-labrax Can you create a PR/change with the new sort? I'll merge it in

Comment (seabass-labrax) [inline joint/2022-06-03.md:21]:
```suggestion
* Alexios Zavras
* Bob Martin
* Dennis Clark
* Dick Brooks
* Gary O'Neall
* Jilayne Lovejoy
* Jim Vitrano
* John Horan
* Karsten Klein
* Marc-Etienne Vargenau
* Mark Atwood
* Matija Suklje
* Michael Herzog
* Phil Odence
* Philippe Ombredanne
* Ria Schalnat
* Sebastian Crane
* Steve Winslow
```

Comment (seabass-labrax) [reply joint/2022-06-03.md:21]: Sure; see below :)

**Review by seabass-labrax — CHANGES_REQUESTED** [review]

File: joint/2022-06-03.md (hunk 1 above)
Comment (seabass-labrax) [reply joint/2022-06-03.md:21]: or see above, at GitHub's whim...!

**Pull request merged**